### DISCUSSION TITLE
optimal alignment points

### DISCUSSION THREAD

**Discussion by ednax** [jomjol/AI-on-the-edge-device#1573] [Q&A] [answered]
Hi,
the wiki says that "The alignment structures needs to be unique and have a good contrast.".
I want to optimize mine, but I'm not sure how this works.
So what form is the perfect one, for the alignment points?

I guess black markings on white (or reverse) is ideal for the contrast.
But what form should I paint? 
Do I need two different forms (Square and circle) or just two identical marks on opposite ends (left and right of the image)?
Or is written Text (like in the most examples) the best?

**Comment by jomjol** [comment, marked as answer]
Well, firstly. Both marks can be the same, just not next to each other (searching are is by default 20 pixel). 
The algo is working as simple autocorrelation (stepping the reference over the pixel and caluclation the average image difference). So hight contrast with clear borders should be benefical.
